polynote-spark/src/main/scala/polynote/kernel/SparkPolyKernel.scala

```diff
@@ -136,7 +136,7 @@ class SparkPolyKernel(
   }
 
   override def init(): IO[Unit] = super.init() >> taskManager.runTaskIO("spark", "Spark session", "Starting Spark session...") {
-    taskInfo => IO(session).handleErrorWith(err => IO(logger.error(err)(err.getMessage)) *> IO.raiseError(err)) >> info.flatMap { update =>
+    taskInfo => IO(session).handleErrorWith(err => IO(logger.error(err)(err.getMessage)) >> IO.raiseError(err)) >> info.flatMap { update =>
```

Comment:
Did you change `*>` to `>>` just for consistency or was it causing issues?

Reply:
A bit of both... I think the meaning of `>>` is usually what we want, and while they're supposed to be the same according to the laws, I think where there are weird issues with execution semantics it's probably better to stick with `>>` so I'm changing them as I see them.